content/documentation/staging/features/index.adoc

```diff
@@ -796,6 +796,26 @@ Kiali has different validations to help troubleshoot configurations related to `
 </div>
 ++++
 
+== Istio Component Status
+
+The Istio service mesh architecture is based in different components: from the istiod to Jaeger. Those components are key for the mesh to work well. That is why Kiali runs regular checks looking for the status of these Istio components.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/istio-components-1.24.png" title="Istio components status: components not healthy or found">
+   <img src="/images/documentation/features/istio-components-1.24.png" style="width: 660px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+There are two possible *statuses* which a component may be in: `Not healthy` or `Not found`. The `Not healthy` status means that the deployment doesn't have the desired amount of replicas running. The `Not found` status means that Kiali is not able to find the deployment.
```

Comment:
There are two possible *statuses* which a component may be in:
-->
A component status will be one of:

Also, a question:  Is there not a healthy status?

Reply:
well, technically there is the healthy status. But it is never shown. I've added it to the list.